YDS-Storybook/SwiftUI/Resources/SwiftUIYDSImageArray.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  SwiftUIYDSImageArray.swift
+//  YDS-Storybook
+//
+//  Created by 심상현 on 2023/09/04.
+//
+
+import SwiftUI
+import YDS_SwiftUI
+
+struct SwiftUIImage {
+    let image: Image?
+    let name: String
+}
```

Comment:
조금 더 직관적인 다른 이름은 어떤가요?
```suggestion
struct ImageDescriptor {
    let image: Image?
    let name: String
}
```

Reply:
음 이부분은 기존 SwiftUI용 IconArray 코드와 유사하게 포맷을 가져가려고 했는데, 음 네이밍을 아예 다르게 가져가는 것도 생각해보면 좋겠군요

Reply:
음 직관적인 이름도 좋긴 한데, 바꾼다고 하면 위에서 얘기했듯이 IconArray 부분도 수정되야할 것 같아서 이건 일단 수정은 안하고 유지하려고 했는데 괜찮을까요?